examples/gallery/embellishments/map_roses.py

```diff
@@ -0,0 +1,149 @@
+r"""
+Directional map roses
+=========
+
+The ``rose`` parameter of the :meth:`pygmt.Figure.basemap` and
+:meth:`pygmt.Figure.coast` methods is used to add a directional map 
+rose to a map. This example shows how such a map rose can be customized:
+
+ - position: **g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**. Set the position
+   of the reference point. Choose from
+
+   - **g**: Give map coordinates as *longitude*\/\ *latitude*.
+   - **j**\|\ **J**: Specify a two-character (order independent) code.
+     Choose from vertical **T**\(op), **M**\(iddle), or **B**\(ottom) and
+     horizontal **L**\(eft), **C**\(entre), or **R**\(ight). Lower /
+     uppercase **j** / **J** mean inside / outside of the map bounding
+     box.
+   - **n**: Give normalized bounding box coordinates as *nx*\/\ *ny*.
+   - **x**: Give plot coordinates as *x*\/\ *y*.
+
+- width: **+w**: Set the width of the rose in plot coordinates (append
```

Comment:
Hm. For me two `:` look not so nice. Maybe we do it like it is for the first item `position` using a period for the second `:`?

```suggestion
- width: **+w**. Set the width of the rose in plot coordinates (append
```